.github/workflows/e2e-tests.yml

```
@@ -177,17 +185,21 @@
 
     - name: Run performance tests
       run: |
-        npx playwright test tests/e2e/performance.spec.js --reporter=json > performance-results.json
-        
-        # パフォーマンス結果の解析
         echo "## Performance Test Results" >> $GITHUB_STEP_SUMMARY
-        if [ -f performance-results.json ]; then
-          # FPS、メモリ使用量、ロード時間などの指標を抽出
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)